FrontEnd/DevPaths/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "./Footer.module.scss";
+
+const Footer = () => {
+  return (
+    <footer className="footer footer-center bg-base-200 text-base-content rounded p-10">
+      <nav className="grid grid-flow-col gap-4">
+        <a href="/" className="link link-hover">
+          Home Page
+        </a>
+        <a href="/" className="link link-hover">
+          RoadMaps
+        </a>
+        <a href="/" className="link link-hover">
+          Contact Us
+        </a>
+      </nav>
+      <nav>
```

Comment:
1. Let's move this social navigation block into a separate component, ensuring the layout is customizable via ` props ` so it can be reused across different parts of the application.
2. Make it dynamically rendered from a list.